Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -1,30 +1,26 @@
 import Foundation
-import Cassette
 
 enum BacktraceBreadcrumbFileHelperError: Error {
     case invalidFormat
 }
 
 @objc class BacktraceBreadcrumbFileHelper: NSObject {
```

Comment:
Does this name sound good for you? It doesnt look like a helper

Reply:
we can change it, drop a class name and I'll rename it.